terraform/stacks/dns/stack.tf

```diff
@@ -97,10 +97,43 @@ resource "aws_route53_record" "cloud_gov_cloud_gov_txt" {
   ttl = 300
   records = [
     "v=spf1 include:mail.zendesk.com ?all",
-    "google-site-verification=JlbE7awT0VhzUvlXbc1wO1yhfFme3lSE1ViUmFX6EEY"
+    "google-site-verification=JlbE7awT0VhzUvlXbc1wO1yhfFme3lSE1ViUmFX6EEY",
+    "zendeskverification.cloud.gov=bb924f6a32697b2b"
```

Comment:
Based on what the zendesk docs said, this needs to be a separate TXT record called `zendeskverification` with the value you have, so this configuration won't work.